.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
         "editor.formatOnSave": true,
         "editor.defaultFormatter": "charliermarsh.ruff",
         "editor.codeActionsOnSave": {
-            "source.organizeImports": true
+            "source.organizeImports": "never"
```

Comment:
Having this on was causing some strange bugs in my vscode where the last line was duplicated every time I saved. So I turned it off.

Reply:
Yes.